sec.club/src/views/Team/Team.scss

```diff
@@ -0,0 +1,83 @@
+@import "../../variables.scss";
+
+body {
+    line-height: 1.8;
+}
+
+h1 {
+    margin-top: 50px
+}
+
+h2 {
+    font-size: 12pt;
+    margin-bottom: 5px;
+    text-align: center;
+    white-space: nowrap;
+}
+
+a{
+    text-decoration: none;
+    color: $text-color;
+    text-align: center;
+    
+}
+
+a:visited {
+    text-decoration: none;
+}
+
+$box-size: 175px;
+
+.profile {
+    width: $box-size;
+    padding: 10px;
+    margin: 10px;
+
+    @media (max-width: $sm-max) {
+        margin: 10px;
+        width: 160px;
+    }
+
+    img {
+        height: $box-size - 20;
+        width: $box-size - 20;
+        border: 1px solid $primary-color;
+        transition: filter 0.25s;
+
+        @media (max-width: $sm-max) {
+            height: 150px;
+            width: 150px;
+        }
+    }
+}
+
+.profile:hover {
+    img {
+        filter: brightness(50%)
```

Comment:
This looks very strange.

Reply:
Should there be no filter on hover? Or something else on hover?

Reply:
I think it'll look better without any filter on hover